core/src/io/Configparser.cpp

```diff
@@ -397,29 +397,29 @@ namespace IO
 					IO::Filter_File_Handle myfile(configFile);
 
 					// N_a
-					myfile.Read_Single(na_left, "pin_na_left", false);
-					myfile.Read_Single(na_right, "pin_na_right", false);
-					myfile.Read_Single(na, "pin_na ", false);
+					myfile.Read_Single(na_left, "pin_na_left", SPIRIT_GENERAL, false);

```

Comment:
Why did you add this specifically to the `Read_Single` reading the pinning values?
Also, you currently create a `Filter_File_Handle` with a file format already, so passing it another should be unnecessary. And when the `Filter_File_Handle` instance used here (`myfile`) is configured correctly with comment strings, this read should work inherently without passing extra info into the function.

Reply:
This is wrong. It was left like that when at first I had the `IO::VF_FileFormat` as argument of `Read_Single()`function. Will be fixed with the next commit.